includes/class-newspack-newsletters.php

```diff
@@ -14,9 +14,10 @@
  */
 final class Newspack_Newsletters {
 
-	const NEWSPACK_NEWSLETTERS_CPT = 'newspack_nl_cpt';
-	const EMAIL_HTML_META          = 'newspack_email_html';
-	const PUBLIC_POST_ID_META      = 'newspack_nl_public_post_id';
+	const NEWSPACK_NEWSLETTERS_CPT     = 'newspack_nl_cpt';
+	const EMAIL_HTML_META              = 'newspack_email_html';
+	const NEWSPACK_NEWSLETTERS_PALETTE = 'newspack_newsletters_color_palette';
```

Comment:
Nit: just to be clear that the variable holds a meta key name and not "a color palette".

```suggestion
	const NEWSPACK_NEWSLETTERS_PALETTE_META_KEY = 'newspack_newsletters_color_palette';
```

Reply:
Fixed in https://github.com/Automattic/newspack-newsletters/pull/1487/commits/346714c7cf3ceded4e47e4e3451658ea6ffe66dd